docs/specs/om/open_metrics_spec_1_1.md

```diff
@@ -635,6 +641,8 @@ foo 18.0 456
 
 The MetricPoint's Total Value Sample MetricName MUST have the suffix `_total`. If present the MetricPoint's Created Value Sample MetricName MUST have the suffix `_created`.
 
+Ingestors MAY accept MetricPoint Total Value name without the `_total` suffix.
```

Comment:
Question to reviewers, a slightly more strict version:
```suggestion
Ingestors MAY accept MetricPoint Total Value name without the `_total` suffix, provided that there is a mechanism to retrieve the counter type information after ingestion.
```

Reply:
That doesn't make sense. We're already only talking about counters, where the TYPE was Counter.

What I'm trying to get at is to protect the user from themselves.

If we allow no suffixes willy-nilly  and don't force/suggest some way to get to the information afterwards, we loose the niceness  of the Prometheus model completely where you could tell from the name that something was a counter. I'd like to at least say "ingestors SHOULD provide the way to retrieve the type information later". For example by `__type__` label or some other means, I don't care.